lib/tasks/data_migration.rake

```diff
@@ -10,6 +10,8 @@ namespace :data_migration do
     subscribers = Subscriber.where(id: subscriptions.pluck(:subscriber_id)).index_by(&:id)
     subscriptions_by_subscriber = subscriptions.group_by(&:subscriber_id).transform_keys { |k| subscribers[k] }
 
+    puts 'hi'
```

Comment:
Some debugging caught behind - or a cryptic message to someone 🙂

Reply:
Yeah, I was investigating a performance problem. It's gone now.